smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -269,21 +269,24 @@ contract IPublicLock is IERC721Enumerable {
     uint amount
   ) external;
 
-  /**
-   * @notice Destroys the msg.sender's key and sends a refund based on the amount of time remaining.
+   /**
+   * @dev Destroys the key managed by msg.sender and sends a refund
+   * based on the amount of time remaining.
    */
-  function cancelAndRefund() external;
+  function cancelAndRefund(uint) external;
```

Comment:
We should include the variable name here as well.

Reply:
done.